app/views/auth_source_ldaps/_ldap_card.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="dropdown pull-right dropdown-kebab-pf">
```

Comment:
The name of the file is `ldap_card` but it only contains the `ldap_card_kebab`.
Would you mind changing the file name?